src/client/Microsoft.Identity.Client/Internal/JsonWebToken.cs

```diff
@@ -205,9 +210,25 @@ public JWTHeaderWithCertificate(ClientCredentialWrapper credential, bool sendCer
 #endif
             }
 
-            [JsonProperty(PropertyName = JsonWebTokenConstants.ReservedHeaderParameters.KeyId)]
+            /// <summary>
+            /// x5t = base64 url encoded cert thumbprint 
+            /// </summary>
+            /// <remarks>
+            /// Mandatory for ADFS 2019
+            /// </remarks>
+            [JsonProperty(PropertyName = JsonWebTokenConstants.ReservedHeaderParameters.X509CertificateThumbprint)]
```

Comment:
As @henrik-me  pointed out, this is a regression introduced by the PoP work (a longer way to say to "introduced by Bogdan" ). I added x5t back and also added kid. Kid is optional but it is added by Wilson - so we are now compliant with Wilson. We have integration tests for AAD which are all passing.